### ISSUE TITLE
Question about CVE-2021-24036 and CVE-2012-2677

### ISSUE THREAD

**Issue by eliw00d** [facebook/flipper#3171]
I am trying to fix the following vulnerabilities that are related to Flipper:

Flipper-Folly: https://www.whitesourcesoftware.com/vulnerability-database/CVE-2021-24036
Flipper-Boost-iOSX: https://www.whitesourcesoftware.com/vulnerability-database/CVE-2012-2677

Are there plans to bump the versions of these to include fixes for these vulnerabilities?

**Comment by mweststrate** [comment]
@lblasa would you be able to bump those deps?

**Comment by lblasa** [comment]
@mweststrate yes, I'm working on it. Unfortunately is not as easy as bumping a version number, but the updates are coming :)

**Comment by ChasePwn14** [comment]
Is there any update on this? We're getting the same flag in our WhiteSource scan after upgrading to the latest version of React Native (0.70.6).

[CVE-2021-24036](https://www.mend.io/vulnerability-database/CVE-2021-24036) found in:

[Flipper-Folly 2.6.10](https://cocoapods.org/pods/Flipper-Folly)

I'm not sure who is in charge of creating a new CocoaPod for Flipper-Folly.
